### DISCUSSION TITLE
`create` method is missing from model

### DISCUSSION THREAD

**Discussion by divmgl** [prisma/prisma#17275] [Q&A]
I've got a model where the `create` method is missing. Nowhere in the documentation does it say why this could be happening. This is a pretty bad blocker. Anyone know why this is happening?

<img width="691" alt="Screenshot 2023-01-12 at 12 39 24 PM" src="https://user-images.githubusercontent.com/5452298/212139638-200f75a7-e284-4afe-8881-cefa92cd8e65.png">

**Comment by nurul3101** [comment]
Hello @divmgl 👋 

Do you have any [Unsupported](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#unsupported) types in the `project_task` model?
If there are any required Unsupported fields then it is expected to not have create and update methods for that specific model.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
